From dfb6b6ac7b8403a37c94e5afb0b990643409cbed Mon Sep 17 00:00:00 2001
From: Andy Yan <andy.yan@rock-chips.com>
Date: Thu, 5 Dec 2024 14:35:48 +0800
Subject: [PATCH] arm64: rockchip_defconfig: Enable DSI2 display

Enable:
CONFIG_ROCKCHIP_DW_MIPI_DSI2=y
CONFIG_PHY_ROCKCHIP_SAMSUNG_DCPHY=y
CONFIG_DRM_PANEL_RAYDIUM_RM67200=y

Signed-off-by: Andy Yan <andy.yan@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index ae949f0148f874..c79513dfe79eb4 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -295,7 +295,6 @@ CONFIG_NVME_TARGET_FC=m
 CONFIG_SRAM=y
 CONFIG_EEPROM_AT24=m
 CONFIG_EEPROM_AT25=m
-CONFIG_EEPROM_93CX6=m
 CONFIG_EEPROM_EE1004=m
 # CONFIG_SCSI_PROC_FS is not set
 CONFIG_BLK_DEV_SD=y
@@ -721,11 +720,13 @@ CONFIG_ROCKCHIP_CDN_DP=y
 CONFIG_ROCKCHIP_DW_HDMI=y
 CONFIG_ROCKCHIP_DW_HDMI_QP=y
 CONFIG_ROCKCHIP_DW_MIPI_DSI=y
+CONFIG_ROCKCHIP_DW_MIPI_DSI2=y
 CONFIG_ROCKCHIP_INNO_HDMI=y
 CONFIG_ROCKCHIP_LVDS=y
 CONFIG_ROCKCHIP_RGB=y
 CONFIG_ROCKCHIP_RK3066_HDMI=y
 CONFIG_DRM_PANEL_RASPBERRYPI_TOUCHSCREEN=m
+CONFIG_DRM_PANEL_RAYDIUM_RM67200=y
 CONFIG_DRM_PANEL_SAMSUNG_S6E88A0_AMS452EF01=m
 CONFIG_DRM_PANEL_SAMSUNG_S6D16D0=m
 CONFIG_DRM_PANEL_SAMSUNG_S6D27A1=m
@@ -965,6 +966,7 @@ CONFIG_PHY_ROCKCHIP_INNO_HDMI=y
 CONFIG_PHY_ROCKCHIP_INNO_USB2=y
 CONFIG_PHY_ROCKCHIP_NANENG_COMBO_PHY=y
 CONFIG_PHY_ROCKCHIP_PCIE=y
+CONFIG_PHY_ROCKCHIP_SAMSUNG_DCPHY=y
 CONFIG_PHY_ROCKCHIP_SAMSUNG_HDPTX=y
 CONFIG_PHY_ROCKCHIP_SNPS_PCIE3=y
 CONFIG_PHY_ROCKCHIP_TYPEC=y
